Jenkinsfile.hardware

```diff
@@ -0,0 +1,133 @@
+#!groovy
+
+pipeline {
+
+    agent {
+
+        node { 
+
+            label 'HardwareSensorBox' 
+
+        }
+
+    }
+
+    environment {
+
+        ANDROID_SDK_ROOT = "/home/catroid/android-sdk-linux"
```

Comment:
As it does not run inside a container, assign the already existing environment variables configured in Jenkins (ANDROID_SDK_LOCATION and ANDROID_HOME) to those variables.